shippable.yml

```diff
@@ -56,7 +56,7 @@ jobs:
       - OUT: aws_vpc_tf_state
     on_success:
       script:
-        - shipctl put_resource_state_multi aws_vpc_tf_info "versionName=$vpc_name" "vpc_id=$(terraform output vpc_id)" "vpc_region=$vpc_region" "vpc_public_sg_id=$(terraform output vpc_public_sg_id)" "vpc_public_sn_id=$(terraform output vpc_public_sn_id)"
+        - shipctl put_resource_state_multi aws_vpc_tf_info "versionName=$vpc_name" "vpc_id=$(terraform output vpc_id)" "vpc_region=$vpc_region" "vpc_public_sg_id=$(terraform output vpc_public_sg_id)" "vpc_public_sn_id=$(terraform output vpc_public_sn_id)" "vpc_memcached_sg_id=$(terraform output vpc_memcached_sg_id)"
```

Comment:
still, i dont like the name memcached security group. Why is this just not a private sg name. why memcache specifi